graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH6.kt

```diff
@@ -0,0 +1,142 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.HasCycle, GraphBehaviours.CreateDirectedGraph, """
```

Comment:
🚫 The first number is the behaviour, the second is the functional element which has that behaviour. It looks like the mapping of functional elements hasn't been populated yet, so you'll need to do that.
https://github.com/anaplan-engineering/azuki/blob/9d03ed017bc2d818533437578ebbe45160ecdb9e/graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/GraphFunctionalElements.kt#L3-L4

Reply:
🚫 ⚠️ I question whether this is a behaviour of directed graphs themselves, though. I would expect there to be a separate "calculation" functional element for determining whether a graph contains cycles or not. It is then arguable whether you would have two behaviours, one for directed and one for undirected graphs, or whether you should have two distinct functional elements with a single behaviour each.